### PR TITLE
feat: add image element

### PR DESCRIPTION
**Pull request by weiijiie**

### CODE REVIEW

File: lib/elements.ts [hunk 1]
```diff
@@ -97,22 +99,31 @@ export function makeDialogue(
...
   return {
     shown: false,
     name: schema.name,
-    position: schema.position || {}, // TODO: settle on a proper default position
+    position: schema.position || {
+      top: '0',
+      right: '0',
+      left: '0',
+      bottom: '0',
+    }, // TODO: settle on a proper default position
```
Comment (tlylt) [inline lib/elements.ts:122]: having all positions of '0' may not be a good default, maybe can take a look at ClickableImg component, I think the setup that I used there is meant to place the image in the center, if no position is specified

File: components/ClickableGroup.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ const ClickableGroup: React.FC<ClickableGroupProps> = ({
   }
 
   return (
-    <div className="absolute left-0 top-0 w-full h-full">
```
Comment (tlylt) [inline components/ClickableGroup.tsx:59]: It used to be required for the positioning of ClickableText or ClickableImg to work well, but if it is ok without it then deleting is fine:)

**Review by tlylt — APPROVED** [review]
Look good to me

File: lib/elements.ts (hunk 1 above)
Comment (weiijiie) [reply lib/elements.ts]: yeah, nice catch, fixed it based on your implementation 👍

File: components/ClickableGroup.tsx (hunk 2 above)
Comment (weiijiie) [reply components/ClickableGroup.tsx:59]: haha yea i'm quite sure it doesn't affect the layout and it tends to make debugging via devtools a little harder as it covers some elements you may be trying to click, so i removed it

**Pull request merged**